clients/java/signalr/src/test/java/HandshakeProtocolTest.java

```diff
@@ -1,6 +1,9 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+import com.microsoft.aspnet.signalr.HandshakeProtocol;
```

Comment:
Need to decide on the ordering of import statements

Reply:
https://google.github.io/styleguide/javaguide.html#s3.3-import-statements

#2676 should be how we resolve this in general.